docs/fields/field-details.asciidoc

```diff
@@ -6434,6 +6469,22 @@ example: `test-pod-cdcws`
 
 // ===============================================================
 
+|
+[[field-orchestrator-resource-parent-type]]
+<<field-orchestrator-resource-parent-type, orchestrator.resource.parent.type>>
+
+| Type or kind of the parent resource of the resource being acted upon.
+
```

Comment:
related to above comment, this would be clearer if "being acted upon" -> "associated with the event"